base/record_replay_inline.h

```diff
@@ -0,0 +1,38 @@
+#if !BUILDFLAG(IS_WIN)
```

Comment:
A comment at the top of the file describing its intended usage (in files that can't call recordreplay:: methods) would be good.